ui/packages/form/src/Dropdown.svelte

```diff
@@ -1,27 +1,139 @@
 <script lang="ts">
-	import { createEventDispatcher } from "svelte";
+    import { fly } from 'svelte/transition';
 	import { BlockTitle } from "@gradio/atoms";
-
+    export let id = '';
+    export let readonly = false;
+    export let placeholder = '';
 	export let label: string;
-	export let value: string | undefined = undefined;
+	export let value: string | Array<string> | undefined = undefined;
```

Comment:
Can't seem to update the value on a multi select drop down

![multiselect_update](https://user-images.githubusercontent.com/41651716/209021859-81c9bd16-8950-4788-8e07-7c356a3d201e.gif)


This is the code I ran

```python

import gradio as gr
import random

with gr.Blocks() as demo:
    dd = gr.Dropdown(["angola", "pakistan", "canada"], multiselect=True, value=["angola"], interactive=True)
    update = gr.Button(value="Update")
    update.click(lambda: gr.Dropdown.update(
                                            value=[random.choice(["angola", "pakistan", "canada"])],
                                            multiselect=True),
                 inputs=None,
                 outputs=[dd])

demo.launch()
```